### PR TITLE
Prevent Civilians from wandering onto Tiberium.

### LINKED ISSUE #18600: Civilians wander onto Tiberium and spawn Visceroids
This random event can give an advantage to one player, or just end the game if it occurs in the first few minutes.

A possible solution is to have civilians avoid Tiberium, as suggested here: #18421

### PR DESCRIPTION
**Pull request by pchote**
This PR implements the simplest fix for #18600. I have resisted the urge to clean up the existing messy code so that we can get this in for the playtest for TDGL.

### CODE REVIEW

**Review by reaperrr — DISMISSED** [review]

File: OpenRA.Mods.Common/Traits/Wanders.cs [hunk 1]
```diff
@@ -95,7 +100,14 @@ CPos PickTargetLocation()
 				if (++ticksIdle % info.ReduceMoveRadiusDelay == 0)
 					effectiveMoveRadius--;
 
-				return CPos.Zero; // We'll be back the next tick; better to sit idle for a few seconds than prolong this tick indefinitely with a loop
+				return null; // We'll be back the next tick; better to sit idle for a few seconds than prolong this tick indefinitely with a loop
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/Wanders.cs]: Style nit: Comments should go on their own line.

File: OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs [hunk 2]
```diff
@@ -28,6 +30,9 @@ class ScaredyCatInfo : TraitInfo, Requires<MobileInfo>
 		[Desc("Chance (out of 100) the unit has to enter panic mode when attacking.")]
 		public readonly int AttackPanicChance = 20;
 
+		[Desc("The terrain types that this actor should avoid running on to while panicing.")]
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs]: `panicking`

**Review by abcdefg30 — COMMENTED** [review]
Not sure if I agree with changing `ScaredyCat`. You wouldn't expect actors panicking to chose a perfectly sane route to do so.

**Comment by pchote** [comment]
> Not sure if I agree with changing ScaredyCat. You wouldn't expect actors panicking to chose a perfectly sane route to do so.

I can certainly understand that perspective, but my testing suggested that `ScaredyCat` is actually the more important change for solving #18600. If we decide not to do this then IMO we might as well WONTFIX the issue (which isn't necessary a bad thing).

**Comment by pchote** [comment]
Rebased and fixed nits.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/8dbfb2da881527f9c1624d1518e181caef264385...0ef1e5c5363aa5972f7f6e55322542d8b4aef72c)

**Pull request merged**